yarn.lock

```diff
@@ -1815,7 +1815,7 @@ colors@1.0.3:
   resolved "https://registry.yarnpkg.com/colors/-/colors-1.0.3.tgz#0433f44d809680fdeb60ed260f1b0c262e82a40b"
   integrity sha1-BDP0TYCWgP3rYO0mDxsMJi6CpAs=
 
-colors@^1.1.2, colors@^1.3.3:
+colors@^1.1.2:
```

Comment:
is it concerning that we still have `colors` in the lockfile? is this from an upstream dep?

Reply:
This is a transitive dependency from `semantic-release` :arrow_right: `marked_terminal` :arrow_right: `cli-table`